python/ray/tune/trial_runner.py

```diff
@@ -489,7 +494,10 @@ def _start_trial(trial: Trial) -> bool:
         if next_trial is not None:
             if _start_trial(next_trial):
                 may_handle_events = False
-            else:
+            elif next_trial.status != Trial.ERROR:
```

Comment:
hmm, could you log an error message if next_trial.status is ERROR (that shouldn't happen)

Reply:
Hm the `trial_executor.start_trial()` will already report an error when this happens. This check is just there to only try to start another actor if the first actor wasn't started due to resources not being ready (e.g. if we have trials with different placement group factories)